mtp_cashbook/assets-src/javascripts/modules/running-total.js

```diff
@@ -0,0 +1,67 @@
+// Running Total module
+/* globals require, exports */
+'use strict';
```

Comment:
Correct. Non-function form should be allowed.
The setting for jshint to stop complaining doesn't seem to be working.